src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleNull.cs

```diff
@@ -71,15 +69,15 @@ private static bool HandleNull(JsonSerializerOptions options, ref Utf8JsonReader
 
             if (!jsonPropertyInfo.IgnoreNullValues)
             {
-                state.Current.JsonPropertyInfo.SetValueAsObject(state.Current.ReturnValue, value: null);
```

Comment:
@steveharter - is it fine to use the local `jsonPropertyInfo` here rather than going back to the state to grab it again? It looks like this should be fine.

Reply:
Yes -- there is no push\pop so the local var is fine.